### ISSUE TITLE
Bungeecoord Crash

### ISSUE THREAD

**Issue by Jesley69** [CrucibleMC/Crucible#103]
I'm having a problem related to the crash in bungeecoord when the player logs in is in another dimension.

Is there any solution for this?

**Comment by spannerman79** [comment]
Use https://github.com/HexagonMC/BungeeCord & Forced hosts.

That or https://github.com/KaiKikuchi/ServerRedirect

**Comment by Jesley69** [comment]
How does "forced host" work? I didn't understand very well, the player doesn't need to log in to the lobby? Is it instantly redirected to the main server?

**Comment by Jesley69** [comment]
I also use "forced hosts" because my server is a cracked server.

**Comment by EverNife** [comment]
If you modpacks are Custom Modpacks, like your own modpacks, the best solution is using the mod ServerRedirect.

If that is not an option, well, mod like ThaumCraft will never work well T.T.

**Comment by spannerman79** [comment]
> If that is not an option, well, mods like ThaumCraft will never work well T.T

Not _quite_.

Using a forced host does work correctly with mods like Thaumcraft.  The issue is a connecting client **must** connect via the forced host and not via bungee/hub then later select the modpacks server via portal/selector/command/etc.

Its the latter that mods like Thaumcraft have issues with.

So this issue is more of a bungee misconfig issue combined with running a server not using correct UUID's.  In short - not a Crucrible issue.

**Comment by Fox2Code** [comment]
Yeah but the problem is switching server, on APOC-Gateway (Private BungeeCord) I successfully fixed it.
Sadly it's not the same story with forge 1.13+, but with forge 1.7.10 it's pretty easy to do by using specific packets.

**Comment by Jesley69** [comment]
How did you fix this in 1.7.10? No way could I.
